libs/core/src/lib/popover/popover-service/popover.service.ts

```diff
@@ -301,16 +301,24 @@ export class PopoverService extends BasePopoverClass {
     /** @hidden */
     private _listenForPositionChange(positionChange: Observable<ConnectedOverlayPositionChange>): void {
         this._placementRefresh$.next();
-        positionChange
-            .pipe(
-                takeUntil(this._placementRefresh$),
-                filter(() => !this.noArrow && !!this._getPopoverBody()),
-                distinctUntilChanged(
-                    (previous, current) =>
-                        previous.connectionPair === current.connectionPair
-                ))
-            .subscribe(event => this._getPopoverBody()._setArrowStyles(event.connectionPair, this._getDirection()))
-        ;
+
+        const observer = positionChange
+            .pipe(takeUntil(this._placementRefresh$));
+
+        // popover cases
+        observer.pipe(
+            filter(() => !this.noArrow && !!this._getPopoverBody()),
+            distinctUntilChanged(
+                (previous, current) =>
+                    previous.connectionPair === current.connectionPair
+            )
+        ).subscribe(event => this._getPopoverBody()._setArrowStyles(event.connectionPair, this._getDirection()));
+
+        // non popover cases, like: combobox, multi-input etc.
+        observer.pipe(
+            filter(() => this.noArrow && !!this._getPopoverBody()),
```

Comment:
not always `noArrow` means combobox or multi-input.
We might have just dropdown as it's shown here:
![image](https://user-images.githubusercontent.com/3263744/131355263-c71bcae8-8491-4b1b-98e1-91bf983cd4e1.png)


But with this change we have this:
![image](https://user-images.githubusercontent.com/3263744/131355147-7bc25dfb-ce0e-43d7-94a2-aa7da6459805.png)

Reply:
We probably need some dedicated input in popover itself to make it clear.
@mikerodonnell89 @platon-rov what do you think?